--- futile-1.8/dicts/dictionaries.f90.orig	2018-12-13 09:27:01.000000000 +0000
+++ futile-1.8/dicts/dictionaries.f90	2021-01-20 12:45:55.942952873 +0000
@@ -197,9 +197,8 @@
 
 
    ! Public variables of the callback module
-   public :: f_err_set_callback,f_err_unset_callback
    public :: f_err_open_try,f_err_close_try
-   public :: f_err_severe,f_err_severe_override,f_err_severe_restore,f_err_ignore
+   public :: f_err_severe,f_err_severe_restore,f_err_ignore
    public :: f_get_past_error,f_get_no_of_errors,f_dump_possible_errors
 
    !for internal f_lib usage
--- bigdft-1.8.3/src/bindings/bindingsf.f90.orig	2018-11-03 10:38:34.000000000 +0000
+++ bigdft-1.8.3/src/bindings/bindingsf.f90	2021-01-20 14:04:22.722722767 +0000
@@ -2003,7 +2003,7 @@
 
 subroutine err_severe_override(callback)
   use f_precisions, only: f_loc
-  use dictionaries, only: f_err_severe_override
+  use exception_callbacks, only: f_err_severe_override
   implicit none
   external :: callback
   
--- bigdft-1.8.3/src/modules/types.f90.orig	2018-12-10 15:41:09.000000000 +0000
+++ bigdft-1.8.3/src/modules/types.f90	2021-01-20 18:25:54.304490025 +0000
@@ -1133,6 +1133,7 @@
   !> Define the BigDFT errors
   subroutine bigdft_init_errors()
     use dictionaries
+    use exception_callbacks, only: f_err_severe_override
     !use module_input_keys, only: input_keys_errors
     implicit none
     external :: bigdft_severe_abort
